.ci/validation.sh

```diff
@@ -1186,6 +1186,63 @@ check-wildcards-on-pitest-target-classes)
   exit "$CLASSES_NO_WILDCARD_COUNT"
   ;;
 
+verify)
+  mvn -e --no-transfer-progress clean verify
+  ;;
+
+package-all-jar)
+  mvn -e --no-transfer-progress clean package -Passembly
+  ;;
+
+website-only)
+  mvn -e --no-transfer-progress clean site -Pno-validations
+  ;;
+
+pmd)
+  mvn -e --no-transfer-progress clean test-compile pmd:check
+  ;;
+
+spotbugs)
+  mvn -e --no-transfer-progress clean test-compile spotbugs:check
+  ;;
+
+checkstyle)
+  mvn -e --no-transfer-progress clean compile antrun:run@ant-phase-verify
+  ;;
+
+forbiddenapis)
+  mvn -e --no-transfer-progress \
+    clean compile test-compile forbiddenapis:testCheck@forbiddenapis-test
+  ;;
+
+test-class)
+  if [[ -z "$2" ]] ; then
+    echo "Error: test class is not defined."
+    echo "Example: mvn -e --no-transfer-progress clean test -Dtest=XdocsPagesTest,XdocsJavaDocsTest"
+    exit 1
+  fi
+  mvn -e --no-transfer-progress clean test -Dtest="$2"
+  ;;
+
+test-method)
+  if [[ -n "$2" ]] ; then
+    echo "Error: test method is not defined."
+    echo "Example: mvn -e --no-transfer-progress clean test -Dtest=CheckerTest#testDestroy"
+    exit 1
+  fi
+  mvn -e --no-transfer-progress clean test -Dtest="$2"
+  ;;
+
+jacoco-local)
```

Comment:
Why is this "local"?

Reply:
We already have `jacoco` in this file.

Reply:
Non-local:
https://github.com/checkstyle/checkstyle/blob/dd4ac5ca90d9f14255c2c7043d423502ca880219/.ci/validation.sh#L1122-L1135

Local:
https://github.com/checkstyle/checkstyle/blob/dd4ac5ca90d9f14255c2c7043d423502ca880219/.ci/validation.sh#L1236-L1240

Reply:
What does `jacoco:check@default-check ` do that it isn't needed in local? Why don't we just have 1 call the other?

Reply:
That is why I told that wiki commands  and CI command  very different and on local you don't need much of such details.

I just do it in continuation of our "just" discussion. I was told we can put all from wiki :).
I need to keep commands simple to let it be shared with not experienced contributors.

Reply:
No merged, as is , with minimal turning, or better to stay at wiki and not even try to mix stuff that should not be mixed.

Reply:
Goal is to let Nick reference command in shell by exact line , not a wiki page.

Reply:
I can rename old jacoco to jacoco-ci

Reply:
> Goal is to let Nick reference command in shell by exact line , not a wiki page.

I am not going to use this, if you are just doing this for me, we can close this PR. The whole point of just is that I can simply type 'just' in the project root, and it tells me each command and exactly what it does. This script will not do that.

Reply:
I use to have a rule: if command in such validation.sh is not in CI , it should be removed, as it might not be actual and might not work.

But I am ok to keep in such files some commands, and yes it might be sometime more convenient to use code and not a wiki.
So I am ok to keep such commands in this file.
Just share with me good names and I will put them.

Reply:
I want this PR to be merged :)

Reply:
I am good with whatever naming that @rnveach suggests

Reply:
> I want this PR to be merged :)

It is still not clear why or who is going to use this,

Reply:
I will use it, now I will know that .ci/validation.sh prints all commands and I can grep

Reply:
`jacoco-local)` is removed.